rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+
+(This means the authors and the community co-own any potential fallout from poorly received posts or videos.)
+
+Blog posts are 500- to 1000-word pieces about a subject, whether that's a single topic, a top N post, a collection of interesting links with commentary, and so on. Some posts will be solidary, and others will be part of a series.
+
+Videos are 3- to 10-minute pieces about a subject, illustrating a narrative that helps those more engaged in visual narratives buy into our concepts and templates.
+
+#### Where content would live
+
+Blog posts would live on a Medium.com magazine, branded as part of the Good Docs Project. Medium.com is a known place where people can subscribe to stories, comment, and otherwise have a sense of expectation regarding a look and feel. I've taken the liberty of (creating a blank publication)[https://medium.com/the-good-docs-project], to make sure we have the name.
+
+Videos would live on YouTube, with icon and naming showing it as part of the Good Docs Project. YouTube is the obvious place, because hosting video content is expensive and YouTube has tools to help with ancillary needs. I've taken the liberty of (creating a blank channel)[https://www.youtube.com/channel/UCi01Sf06CeUeW5XlwqyNIpQ].
+
+*Aside:* We understand there's a blog component to the GDP website. We think that may be a good place to explore hosting white papers, to keep white papers directly attached to the material they support.
+
+Regarding external housing: It's the responsibility of the publishers to make the Terms of Service of those services known to the Project Steering Committee. That includes any changes we're aware of.
+
+* [Medium's Terms of Service](https://policy.medium.com/medium-terms-of-service-9db0094a1e0f) as of 9/12/2020
+* [YouTube's Community Guidelines](https://www.youtube.com/howyoutubeworks/policies/community-guidelines/) help center
+
+### Organization & personnel
+
+First, Procope ideally operates on a "fail early, learn, keep on keeping on." Keep that in mind.
+
+Publication standards are important to all organizations, especially to those promoting publication standards themselves. To that end, there's a hierarchical model borrowed from Ryan's periodical background.
+
+* There's a central **publisher** role, held by two people. They have publication rights on the external accounts (Medium, YouTube). They maintain the publication queue and quality, and have as part of their role to also recruit contributors.
+	* We want to encourage publisher rotation. This could mean one publisher steps down around the time of PSC transitions (around March) or six months afterward, allowing an interested editor to fill that role. This means part of being an editor is getting the specific experience needed to take on the role of publisher. (The first handoff may be bumpy, so we'll play this by ear.)
+	* First publishers would be: Ryan Macklin & Carrie Crowe.
```

Comment:
**Comment (non-blocking):** Maybe consider changing out your publishers on a timeline that isn't necessarily related to the PSC's timeline. As far as I'm aware, PSC changes have no official timeline for changing in and out members. Someone simply makes a nomination or announces they are stepping down.

Reply:
That was my original thought. I'm going to remove the specific timeline element for now, and put that in a "we'll review every six months after the blog starts the state of the project and the health of contributor bandwidth."